README.md

```diff
@@ -30,6 +30,7 @@ This project uses the following third party crates:
 * cc (MIT and Apache-2.0)
 * log (MIT and Apache-2.0)
 * serde (MIT and Apache-2.0)
+* walkdir (MIT or UNLICENSE)
```

Comment:
Eh... is UNLICENSE compatible with Apache-2.0? 

I'm not sure I understand very well if it leans the GPL way: https://unlicense.org/

Reply:
Sounds like it's meant to be compatible with (almost?) any license - they just renounce their rights over the code altogether, so should be fine, but maybe we need to check

Reply:
Got the confirmation it was ok to use.